Fails to Install on RHEL Based Distros

## Expected Behaviour

I should be able to use the Go SDK on RHEL based distributions.

The following code should return an identity:

```
package main

import "fmt"
import "github.com/joinself/self-go-sdk"

func main() {
  cfg := selfsdk.Config{
    SelfAppID:           "<app-id>",
    SelfAppDeviceSecret: "<app-secret-key>",
    StorageDir:          "/data",
    StorageKey:          "secret",
    Environment:         "review",
  }

  client, _ := selfsdk.New(cfg)
  client.Start()

  svc := client.IdentityService()

  identity, _ := svc.GetIdentity("56653660996")

  fmt.Println(identity)
}
```

## Actual Behaviour

```
[root@529b95d5def3 wd]# go run test.go 
go: downloading github.com/joinself/self-go-sdk v0.0.0-20230621110528-a4a56824fe03
go: downloading golang.org/x/crypto v0.0.0-20200323165209-0ec3e9974c59
go: downloading github.com/joinself/self-crypto-go v0.0.0-20230620152847-d5b4947a8e3c
go: downloading github.com/mattn/go-sqlite3 v1.14.17
go: downloading github.com/google/uuid v1.1.1
go: downloading github.com/google/flatbuffers v2.0.0+incompatible
go: downloading github.com/gorilla/websocket v1.4.1
go: downloading github.com/lucasb-eyer/go-colorful v1.0.3
go: downloading github.com/skip2/go-qrcode v0.0.0-20191027152451-9434209cb086
go: downloading github.com/tidwall/gjson v1.9.3
go: downloading github.com/square/go-jose v2.4.1+incompatible
go: downloading github.com/tidwall/sjson v1.0.4
go: downloading google.golang.org/protobuf v1.25.0
go: downloading gopkg.in/square/go-jose.v2 v2.4.1
go: downloading github.com/beevik/ntp v0.2.0
go: downloading github.com/tidwall/match v1.1.1
go: downloading github.com/tidwall/pretty v1.2.0
go: downloading golang.org/x/sys v0.5.0
go: downloading golang.org/x/net v0.0.0-20200222125558-5a598a2470a0
# command-line-arguments
/usr/lib/golang/pkg/tool/linux_amd64/link: running gcc failed: exit status 1
/usr/bin/ld: cannot find -lstdc++: No such file or directory
/usr/bin/ld: cannot find -lstdc++: No such file or directory
/usr/bin/ld: cannot find -lstdc++: No such file or directory
/usr/bin/ld: cannot find -lstdc++: No such file or directory
collect2: error: ld returned 1 exit status
```

## Additional Information

Similar issue to: https://zube.io/joinself/self/c/11578

--- a/README.md
+++ b/README.md
@@ -22,13 +22,14 @@ This SDK provides access to the following self services:
 
 ##### Debian/Ubuntu
 ```bash
-curl -LO https://github.com/joinself/self-omemo/releases/download/0.4.0/self-omemo_0.4.0_amd64.deb
-apt install -y ./self-omemo_0.4.0_amd64.deb
+curl -LO https://github.com/joinself/self-omemo/releases/download/0.5.0/self-omemo_0.5.0_amd64.deb
+apt install -y ./self-omemo_0.5.0_amd64.deb
 ```
 
 ##### CentOS/Fedora/RedHat
 ```bash
-rpm -Uvh https://github.com/joinself/self-omemo/releases/download/0.4.0/self-omemo-0.4.0-1.x86_64.rpm
+yum install -y libstdc++-static
+rpm -Uvh https://github.com/joinself/self-omemo/releases/download/0.5.0/self-omemo-0.5.0-1.x86_64.rpm
 ```
 
 ##### MacOS - AMD64

